status.py

```diff
@@ -0,0 +1,7 @@
+NOT_STARTED = 'Not Started'
```

Comment:
Again you could move this a class with class attributes (which would essentially make Status a namespace)